java/runtime/src/main/java/org/ray/runtime/Worker.java

```diff
@@ -89,11 +89,15 @@ public void execute(TaskSpec spec) {
       // Get local actor object and arguments.
       Object actor = null;
       if (spec.isActorTask()) {
+        // TODO(qwang): Remove `worker.currentActorId`,
+        // and use `runtime.currentActorId` instead.
+        runtime.setCurrentActorId(spec.actorId);
```

Comment:
Better still put `currentActorId` in worker. and `RuntimeContext` can just use `runtime.woker.currentActorId`